webapp/src/store/users.ts

```diff
@@ -19,11 +19,12 @@ type UsersStatus = {
     me: IUser|null
     workspaceUsers: {[key: string]: IUser}
     loggedIn: boolean|null
+    userWorkspaces: UserWorkspace[]
```

Comment:
I would say that this belong to `workspace.ts` not to `users.ts`. `users.ts` handles the users models, in this case, return myself, and the list of users. Whenever you need workspace instances, I expect them to be in `workspace.ts`.

Reply:
Done.